### DISCUSSION TITLE
ultima-ng-17.0.0 panelmenu menuitem badge are not displayed

### DISCUSSION THREAD

**Discussion by cydriclopez** [primefaces/community#1435] [PrimeNG Templates]
Hi,

I am using primeng ultima-ng-17.0.0 and now in panelmenu the menuitem badge do not show anymore. In the previous ultima-ng-15.0.1 they did show. What has changed? Am I missing something?

Thanks

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a code example and screen recording if possible, so we can identify the issue?

**Reply by cydriclopez** [reply]
@cetincakiroglu good day to you. Thanks for the prompt reply.

I hope this gives you enough info. I took screenshots below. Note that in ultima-ng-15.0.1 the badge "1" is displayed. Compare this to ultima-ng-17.0.0 where the badge does not show. I pasted same exact code below.

Any help is appreciated. For sometime I focused on my transition from postgres to Cassandra. Now I am back on the client side this is the first thing I noticed. Again thanks much.

ultima-ng-15.0.1
![ultima-ng-15 0 1](https://github.com/primefaces/community/assets/8703893/32e256e6-0967-4385-b71f-b1b8f91728ff)

ultima-ng-17.0.0
![ultima-ng-17 0 0](https://github.com/primefaces/community/assets/8703893/b66ded64-0c4e-40c1-a614-218a03adc9b6)

```ts
ngOnInit(): void {
    this.items = [
        {
        label: 'File',
        icon:'pi pi-fw pi-file',
        items: [
            {
                label: 'New',
                icon:'pi pi-fw pi-plus',
                items: [
                      {
                          label: 'Bookmark',
                          icon:'pi pi-fw pi-bookmark',
                          styleClass: 'p-badge-warning',
                          title: 'Click for bookmark',

                          // *************************************************************
                          // In ultima-ng-17.0.0 panelmenu menuitem badge are not displayed
                          badge: '1',
                          badgeStyleClass: 'p-badge-warning'
                          // *************************************************************

                      },
                      {
                          label: 'Video',
                          icon:'pi pi-fw pi-video',
                      }
                ]
            },
            {
                label: 'Delete',
                icon:'pi pi-fw pi-trash',
            },
            {
                label: 'Export',
                icon:'pi pi-fw pi-external-link',
            }
        ]
        /*
        (snipped entries)
        .
        :
        */
        }
    ]
}
```

**Reply by cetincakiroglu** [reply]
Hi,

Thanks for the explanation and the code example, we'll investigate and get back to you.
